Remove exception strack trace print when loading class

johnrengelman opened the pull request:
Erik -
another quick clean up to the _Events.groovy script. Just removes an exception print stack trace that is unnecessary. No need to release an update.

-John

johnrengelman committed:
Remove stack trace print when unable to load class.

bodiam commented:
Hi John,

Thanks, I'll merge it!

Erik

On 7 mei 2012, at 19:59, John Engelman reply@reply.github.com wrote:

> Erik -
> another quick clean up to the _Events.groovy script. Just removes an exception print stack trace that is unnecessary. No need to release an update.
> 
> -John
> 
> You can merge this Pull Request by running:
> 
>  git pull https://github.com/johnrengelman/grails-fitnesse master
> 
> Or you can view, comment on it, or merge it online at:
> 
>  https://github.com/bodiam/grails-fitnesse/pull/5
> 
> -- Commit Summary --
> - Remove stack trace print when unable to load class.
> 
> -- File Changes --
> 
> M scripts/_Events.groovy (2)
> 
> -- Patch Links --
> 
>  https://github.com/bodiam/grails-fitnesse/pull/5.patch
>  https://github.com/bodiam/grails-fitnesse/pull/5.diff
> 
> ---
> 
> Reply to this email directly or view it on GitHub:
> https://github.com/bodiam/grails-fitnesse/pull/5

johnrengelman committed:
Remove adding fitnesse phase to phasesToRun in eventAllTestsStart since it can't be overriden by the application.

johnrengelman commented:
Erik -
Ran into another issue today and I updated the pull request with the new commit. Looks like we are forcing the "fitnesse" phase into the test phases to run in the _Events.groovy, however this forces the phase to run regardless of any manipulation of phasesToRun by the parent application. Which means that I always have to run the fitnesse phase...not good. So I removed the line that forces the fitnesse phase. I think we should update the documentation to reflect that the application will need to specify inclusion of the fitnesse phase in their _Events.groovy.

This one I think necessitates a new release.

Thanks again!

John

johnrengelman commented:
Hey Erik -
Just wanted to check on status here or if you had any questions. Thanks!

John
